internal/services/containers/kubernetes_fleet_manager_resource_test.go

```diff
@@ -51,43 +46,13 @@ func TestAccKubernetesFleetManager_requiresImport(t *testing.T) {
 func TestAccKubernetesFleetManager_complete(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_kubernetes_fleet_manager", "test")
 	r := KubernetesFleetManagerTestResource{}
-
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
 			Config: r.complete(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep("hub_profile.#", "hub_profile.0.%", "hub_profile.0.dns_prefix", "hub_profile.0.fqdn", "hub_profile.0.kubernetes_version"),
-	})
-}
-
-func TestAccKubernetesFleetManager_update(t *testing.T) {
```

Comment:
Please keep this test case to verify if the `tags` can be updated. You can create a `update` function to provide the config with updated `tags`.